modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Radio/Radio.es.js

```diff
@@ -64,6 +68,14 @@ const Radio = ({
 						setCurrentValue(value);
 						onChange({target: {value}});
 					}}
+					onKeyUp={(event) => {
+						const value = options[0].value;
+
+						if (!currentValue && event.keyCode === KEYCODES.TAB) {
+							setCurrentValue(value);
+							onChange({target: {value}});
+						}
+					}}
```

Comment:
if `(!currentValue && event.keyCode === KEYCODES.TAB)` evaluates to false, then the value assignment is useless. I'd just move to the if body:

```
onKeyUp={(event) => {
	if (!currentValue && event.keyCode === KEYCODES.TAB) {
		const value = options[0].value;

		setCurrentValue(value);
		onChange({target: {value}});
	}
}}
```

Reply:
hey @cgmonte, is great to have you back! :blush: what you said make sense, i will change and send again, thanks!